### DISCUSSION TITLE
Combobox on Modal Issue

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2977] [Q&A] [answered]
We are having an issue where the combobox dropdown height is restricted to the modal size--is this an issue with our implementation, or is this supposed to happen by definition? I noticed that [examples on the Modal page](https://paste.twilio.design/components/modal/#setting-initial-focus) were using the select component instead of combobox--wondering if this is a workaround?
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/87049408/214373960-7a4e3918-c253-4679-92fb-b90f29a3fd8c.png">
<img width="639" alt="image" src="https://user-images.githubusercontent.com/87049408/214374363-0a9ff0bf-8f61-4172-9274-deb92dccae10.png">

**Comment by loreina** [comment]
Hi @rosemaryluo, thanks for catching this. We've merged #2949 for this fix already and expect to publish version packages with the fix included in the next couple of days.

**Comment by shleewhite** [comment, marked as answer]
Hi @rosemaryluo!

The fix is now available in `@twilio-paste/combobox` to `13.0.3` or `@twilio-paste/core` to `17.0.5`.

Let me know if you continue to have issues! Thank you again for letting us know about this bug.

**Reply by rosemaryluo** [reply]
I'm still seeing the behavior on our live page, do I need to ask my team to update something or should this be immediately reflected?

**Reply by gloriliale** [reply]
hi @rosemaryluo! The team will need to update to those versions of the packages, then it should be available. If it still crops up after they've updated, let us know!
